src/dsp_tools/xmllib/models/dsp_base_resources.py

```diff
@@ -53,6 +56,17 @@ def add_comments(self, comments: list[str]) -> AnnotationResource:
         self.comments.extend(comments)
         return self
 
+    def add_migration_metadata(
```

Comment:
It is rare but Rita requested it, which is why I did not put it into the `new` function.
I plan on asking Noemi about these stylistic choices, she is closer to RDU than me, but will be able to give an opinion without seeing the options perfectly fleshed out.

Reply:
unfortunate that we need to add it... but I understand that it's a requirement.

how would you feel about adding the `@deprecated("Only used for Salsah migration. Will be removed eventually.")` decorator to it right away, so that no misunderstandings arise?

Reply:
(of course this goes for all of those, I'm not commenting on each of them)